deployments/goerli/usdc/relations.ts

```diff
@@ -9,5 +9,17 @@ export default {
         field: async (fxRoot) => fxRoot.stateSender()
       }
     }
+  },
+  'inbox': {
```

Comment:
I notice this is missing the relations for `l1GatewayRouter`. Is that intentional?